pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
```

Comment:
Please fix import to correctly use Button and ensure it’s consistently referenced as Button in the file.

```suggestion
import { Button } from "@/components/Button";
```